test/pytest/test_binary_cnn.py

```diff
@@ -73,5 +90,4 @@ def test_model2(backend, io_type):
     y = model2.predict(X)  # noqa: F841
     y_hls = hls_model.predict(X)  # noqa: F841
 
-    # # TODO:  enable the comparions after fixing the remaing issues
-    # np.testing.assert_allclose(np.squeeze(y_hls), np.squeeze(y), rtol=1e-2, atol=0.01)
+    np.testing.assert_allclose(np.squeeze(y_hls), np.squeeze(y), rtol=1e-2, atol=0.01)
```

Comment:
The old version of this test seems to have had one data point to verify functionality between QKeras and hls4ml - can that be increased, to maybe a 100? Increasing also removes the need for `np.squeeze`.

Reply:
Agree.